src/app/session/[type]/Steps/Basic/index.tsx

```diff
@@ -43,6 +44,7 @@ const BasicForm: FC = () => {
         label: 'Platform',
         disabled: type === SessionType.EDIT,
         onValueChange: (value, form) => handleBatchFields(value, form, apiOptions, fieldsSchema),
+        helperText: `Sessions where platform is "${Platform.Others}" are listed under "Live Classes" tab.`,
```

Comment:
i think the right statement is: sessions where platform is anything other than "quiz" is listed in liveclasses tab. check for grammar here

Reply:
Rephased it to this one.